ui/webui/src/components/AnacondaWizard.jsx

```diff
@@ -330,8 +334,11 @@ const Footer = ({
         <WizardFooter>
             <WizardContextConsumer>
                 {({ activeStep, onNext, onBack }) => {
+                    const isFirstScreen = (
+                        activeStep.id === "installation-language" || (activeStep.id === "storage-devices" && isLiveOS)
+                    );
                     const isBackDisabled = (
-                        activeStep.id === "installation-language"
+                        isFirstScreen
                     );
```

Comment:
You can actually have just `const isBackDisabled = isFirstScreen;`. So then - do we need both `isBackDisabled` and `isFirstScreen` when only one is used later in the code...?